core/translate/expr.rs

```diff
@@ -1942,6 +1952,34 @@ pub fn translate_expr(
     }
 }
 
+// Returns the starting register for the function.
+// TODO: Use this function for all functions with variable number of parameters in `translate_expr`
+fn translate_variable_sized_function_parameter_list(
+    program: &mut ProgramBuilder,
+    args: &Option<Vec<ast::Expr>>,
+    referenced_tables: Option<&[BTreeTableReference]>,
+    precomputed_exprs_to_registers: Option<&Vec<(&ast::Expr, usize)>>,
+) -> Result<usize> {
+    let args = args.clone().unwrap_or_default();
```

Comment:
I changed `unwrap_or(Vec::new())` to `unwrap_or_default()` because Clippy suggested that, but IMO it's less obvious now 🙈